src/main/java/io/jsonwebtoken/BadAudienceJwtException.java

```diff
@@ -0,0 +1,11 @@
+package io.jsonwebtoken;
```

Comment:
Really minor thing here, current classes in the library have the Apache license in it.